packages/react-components/src/components/SendBox.tsx

```diff
@@ -305,18 +305,18 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
   }, [props.activeFileUploads, fileUploadsPendingError]);
 
   /* @conditional-compile-remove(file-sharing) */
-  const onRenderFileUploads = useCallback(
-    () =>
-      props.onRenderFileUploads ? (
-        props.onRenderFileUploads()
-      ) : (
-        <_FileUploadCards
-          activeFileUploads={props.activeFileUploads ? props.activeFileUploads : []}
-          onCancelFileUpload={props.onCancelFileUpload}
-        />
-      ),
-    [props]
-  );
+  const onRenderFileUploads = useCallback(() => {
+    if (!props.activeFileUploads?.length) {
+      return null;
+    }
+    return props.onRenderFileUploads ? (
+      props.onRenderFileUploads()
+    ) : (
+      <div style={{ margin: '0.25rem' }}>
```

Comment:
Looks like we should avoid setting this margin when there are active file uploads, but they failed (or there is some other logic needed to remove them from props.activeFileUploads):
![image](https://user-images.githubusercontent.com/2684369/162014594-818bcbb4-1f4d-4e81-a1af-07aab4d998e4.png)

Reply:
Fixed, thanks for pointing out.